src/utils/sanitizePackageName.js

```diff
@@ -0,0 +1,5 @@
+const unsafeRegex = /[/*]/g;
+
+module.exports = function sanitizePackageName(name, replacement = '-') {
```

Comment:
Could make this a more generic `sanitizeFileName` so that it can be used elsewhere if an issue presents itself.